code/modules/surgery/other.dm

```diff
@@ -1,4 +1,4 @@
-//Procedures in this file: Inernal wound patching, Implant removal.
+//Procedures in this file: Inernal wound patching, Implant removal, Hardsuit removal.
```

Comment:
unrelated? I don't see any changes to surgery here

Reply:
You're right